integration/docker/Dockerfile.fuse

```diff
@@ -24,14 +24,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends software-proper
 
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 
-RUN mkdir -p /alluxio-fuse
-
 ADD ${ALLUXIO_TARBALL} /opt/
 
 # if the tarball was remote, it needs to be untarred
+# use ln -s instead of mv to avoid issues with Centos (see https://github.com/moby/moby/issues/27358)
```

Comment:
Can we remove `ENV ENABLE_FUSE true` from above now?

Reply:
yes